cpp/src/arrow/compute/api_scalar.h

```diff
@@ -117,6 +119,18 @@ class ARROW_EXPORT RoundTemporalOptions : public FunctionOptions {
   CalendarUnit unit;
   /// What day does the week start with (Monday=true, Sunday=false)
   bool week_starts_monday;
+  /// If True times exactly on unit multiple boundary will be rounded up one unit.
+  /// This applies for ceiling only.
```

Comment:
Suggestion from a chat: "Enable this flag to return a rounded value that is strictly greater than the input", and then give one example to clarify further

Reply:
Done.